findbugs/build.gradle

```diff
@@ -213,11 +130,12 @@ distributions {
 
         // TODO : These exclusions can be removed when we move dependencies from lib to dependency management
         exclude 'AppleJavaExtensions.jar'
+        exclude 'hamcrest-all-1.3.jar'
```

Comment:
It comes from Lin directory.
https://github.com/spotbugs/spotbugs/tree/47d9a90d697542d1cef48a7a0d4104d6b128817d/findbugs/lib

Refs: f038a08505ed8dcdf6d1b85cccc371f6b69132a6